### ISSUE TITLE
Task name showing as story name

### ISSUE THREAD

**Issue by Fej3000** [figueiredorui/1-click-child-links#107]
Hi 
When adding tasks to a user story the tasks are created but they all have the story name as there name. They are not created with the names they were given when created in the template. E.G. user story named "1click test", all tasks created via 1-click-child-links are named "1click test" and not the names they were saved as in the templates section of task1 , task2 ...

Has any one encountered this?

**Comment by odonovan-johnd** [comment]
Yes, I am seeing this today. Just installed extension today.

**Comment by medha-bhatt** [comment]
Yes, I too am seeing the same issue - is there any fix available yet?

**Comment by byronweiss** [comment]
I think this happens if the task has no title entered into it.  Can be confusing because `title` is actually a field you need to add/select.

![image](https://user-images.githubusercontent.com/102189859/179087556-abb2407f-005e-432e-9af2-7938fecd168d.png)

**Comment by medha-bhatt** [comment]
@byronweiss  No, that's not the case - I am entering all the fields properly.   Even then, this issue occurs.

**Comment by jeanblakeleycook** [comment]
I was able to resolve the problem as described by bryonweiss above. I simply added the field, Title, and named it what i wanted the title of the task to be (e.g., 1. Acceptance Criteria Review) and it worked beautifully. I suggest you give it a go!
